server/src/main/java/com/datastrato/gravitino/server/web/rest/PartitionOperations.java

```diff
@@ -50,15 +53,21 @@ public Response listPartitionNames(
       @PathParam("metalake") String metalake,
       @PathParam("catalog") String catalog,
       @PathParam("schema") String schema,
-      @PathParam("table") String table) {
+      @PathParam("table") String table,
+      @QueryParam("verbose") @DefaultValue("false") boolean verbose) {
```

Comment:
I would suggest to change the quey parameter to "details".

Reply:
ok